packages/iTwinUI-react/src/core/Wizard/WorkflowDiagram.tsx

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/workflow-diagram.css';
+import { WizardProps } from './Wizard';
+import { Step } from './Step';
+
+export type WorkflowDiagramProps = Pick<WizardProps, 'steps'>;
```

Comment:
I think this should either define its own props or `Pick<>` props from `Stepper` when `Wizard`'s props are moved to `Stepper` like bentleyvk suggested in Stepper.tsx.

Reply:
Both Wizard and WorkflowDiagram props are now picked from Stepper